### ISSUE TITLE
QUERY/files command does not support UseFileIgnore=false

### ISSUE THREAD

**Issue by szampier** [ICRAR/ngas#98]
The query command does not support the old `ignore` column in `ngas_files`
Example: `curl http://<host>:<port>/QUERY?query=files_list`
Error: Invalid column name 'file_ignore'

**Comment by rtobar** [comment]
@szampier could you try the fix I've just pushed? It's in the `98-query-ignore` branch. I don't have a database handy with that column name, but I believe the change should work. I could also investigate if I can run the test suite against an "ignore"-enabled database instear of a "file_ignore" one, but that might be a bit more of work (or not, would need to dedicate some time to investigate).

**Comment by szampier** [comment]
The fix works if you remove the following line which is probably a leftover and overwrites the replacements:
https://github.com/ICRAR/ngas/blob/8d398981f6ad55691fe1c091a551bd2dc3b293a9/src/ngamsServer/ngamsServer/commands/query.py#L62

Also the default argument `replacements={}` is mutable (warning in Pycharm) and should be handled differently although in this case it's probably not a big issue. I've seen this issue in other parts of the code.

Lastly, `files_list_recent` doesn't work on Sybase because `limit` is not a standard SQL (I think) but probably there isn't much we can do here.

**Comment by rtobar** [comment]
Ah, thanks, that is indeed a leftover, I've removed it and force-pushed the branch. I also took the change of rebasing on top of the latest master, where I've finally merged the move from Travis CI to GitHub Actions for our automatic checks -- Travis has been giving too many problems, and in fact tests are were not running at the moment.

I'm fully aware the default argument isn't ideal as written, but I'm not too fuzzed about it here, where the function is very local, and the argument is used only to read data, so I consciously made the decision to keep it that way to keep things small. I've indeed seen the same around the code here and there, but I probably won't proactively go out looking for those unless they are causing issues.

> Lastly, files_list_recent doesn't work on Sybase because limit is not a standard SQL (I think) but probably there isn't much we can do here.

Yeap, `OFFSET` seems to be in a pretty dire situation indeed: https://stackoverflow.com/questions/1528604/how-universal-is-the-limit-statement-in-sql
